routes/w3s.ts

```diff
@@ -26,19 +26,19 @@ async function parseProof(data) {
 }
 
 export default defineEventHandler(async (event) => {
-  const requestObject = event.node.req
+  const tokenData = event.context.authTokenData
```

Comment:
Why are we doing an auth check again here? Isn't the middleware supposed to be doing this?